pkg/repository/v1/sqlcv1/filters.sql

```diff
@@ -91,7 +91,7 @@ GROUP BY workflow_id
 
 -- name: ListFiltersForEventTriggers :many
 WITH inputs AS (
-    SELECT
+    SELECT DISTINCT
```

Comment:
deduping here to handle the case where there are duplicate id, scope pairs which would cause the join in the main query to blow up